client/src/components/AddCard/AddCardInfo/AddCardInfoForm/AddCardInfoForm.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { useStripe, useElements, PaymentElement } from '@stripe/react-stripe-js';
+
+const SetupForm = () => {
+  const stripe = useStripe();
+  const elements = useElements();
+
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const handleSubmit = async (event: { preventDefault: () => void }) => {
+    event.preventDefault();
+
+    if (!stripe || !elements) {
+      return;
+    }
+
+    const { error } = await stripe.confirmSetup({
+      elements,
+      confirmParams: {
+        return_url: process.env.REACT_APP_DOMAIN + '/dashboard',
+      },
+    });
+
+    if (error.message) {
+      setErrorMessage(error.message);
+    } else {
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <PaymentElement />
+      <button disabled={!stripe}>Submit</button>
+      {errorMessage && <div>{errorMessage}</div>}
```

Comment:
Please use MUI `Button` component for `<button>` and use `Box` component for `<div>`

Reply:
It is the Stripe component. I was not sure should I change any code in it. Thank you for your comment. Let's change it.